script/CalculateSafeHashes.s.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+import "forge-std/Script.sol";
+import {GnosisSafeHashes} from "src/libraries/GnosisSafeHashes.sol";
+
+contract CalculateSafeHashes is Script {
```

Comment:
Who calls this? What's the `TENDERLY_PAYLOAD`? I know its mentioned in the `simulate-verify-task.sh` file below but more information in this file as to what it's purpose is would be helpful.

Reply:
Good point, I added a detailed comments about that.